lib/peddler/client.rb

```diff
@@ -38,6 +38,10 @@ def path(path = nil)
         path ? @path = path : @path ||= '/'
       end
 
+      def version(version = nil)
+        version ? @version = version : @version ||= ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.